src/js/samples/components/Item/Item.tsx

```diff
@@ -0,0 +1,201 @@
+        margin: 0;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
+`;
+
+const StyledSampleItem = styled(Box)`
+    align-items: stretch;
+    display: flex;
+    flex-basis: 0px;
+`;
+
+type SampleItemProps = {
+    id: string;
+    index: number;
+    checked: boolean;
+    ready: boolean;
+    name: string;
+    created_at: string;
+    handle: string;
+    job?: JobMinimal;
+    library_type: LibraryType;
+    labels: Array<LabelNested>;
+    workflows: SampleWorkflows;
+    onSelect: (id: string, index: number, shiftKey: boolean) => void;
+    onQuickAnalyze: (id: string) => void;
+};
+
+/**
+ * A condensed sample item for use in a list of samples
+ *
+ * @param id - The unique identifier of the sample
+ * @param index - Index of the sample
+ * @param checked - Whether the sample is selected
+ * @param handle - The user who created the sample
+ * @param created_at - The date the sample was created
+ * @param name - Name of the sample
+ * @param library_type - Library type associated with the sample
+ * @param ready - Whether the sample is ready
+ * @param workflows - Workflows associated with the sample
+ * @param labels - Labels associated with the sample
+ * @param job - Information about the job associated with the sample
+ * @param onSelect - Callback to handle sample selection
+ * @param onQuickAnalyze - Callback to trigger quick analysis
+ */
+export function SampleItem({
+    id,
+    index,
+    checked,
+    handle,
+    created_at,
+    name,
+    library_type,
+    ready,
+    workflows,
+    labels,
+    onSelect,
+    onQuickAnalyze,
+}: SampleItemProps) {
+    const handleCheck = e => {
+        onSelect(id, index, e.shiftKey);
+    };
+
+    const handleQuickAnalyze = () => {
+        onQuickAnalyze(id);
+    };
+
+    return (
+        <StyledSampleItem>
+            <SampleItemCheckboxContainer onClick={handleCheck}>
+                <Checkbox checked={checked} />
+            </SampleItemCheckboxContainer>
+
+            <SampleItemData>
+                <SampleItemMain>
+                    <SampleItemTitle>
+                        <Link to={`/samples/${id}`}>{name}</Link>
+                        <Attribution time={created_at} user={handle} />
+                    </SampleItemTitle>
+                </SampleItemMain>
+                <SampleItemLabels>
+                    <SampleLibraryTypeLabel libraryType={library_type} />
+                    {labels.map(label => (
+                        <SmallSampleLabel key={label.id} {...label} />
+                    ))}
+                </SampleItemLabels>
+            </SampleItemData>
+            <SampleItemWorkflows>
+                <WorkflowTags id={id} workflows={workflows} />
+            </SampleItemWorkflows>
+            <SampleItemIcon>
+                <EndIcon ready={ready} onClick={handleQuickAnalyze} />
+            </SampleItemIcon>
+        </StyledSampleItem>
+    );
+}
+
+export function mapStateToProps(state, ownProps) {
```

Comment:
Do these have tests? If not, remove the `export` as they aren't needed outside the module.